### PR TITLE
add theme screenshot to search results

### PR DESCRIPTION
**Pull request by propertunist**
this is fix for #29

### CODE REVIEW

File: views/default/object/plugin_project/search.php [hunk 1]
```diff
@@ -10,6 +10,14 @@
 	'encode_text' => TRUE,
 ));
 
+$img_files = $project->getScreenshots();
+
+if ($img_files) {
+	$thumb = get_entity($img_files[0]->thumbnail_guid);
+	$src = elgg_get_site_url() . "plugins/icon/{$img_files[0]->getGUID()}/icon.jpg";
+	$preview_img = "<a href=\"$project->getURL()\" class=\"plugin_preview\"><img src=\"$src\" alt=\"$project->title\" title=\"$project->title\" width=\"150\"/></a>";
```
Comment (mrclay) [inline views/default/object/plugin_project/search.php]: Use elgg_format_element(). Makes sure escaping happens and it's cleaner

Comment (propertunist) [inline views/default/object/plugin_project/search.php]: i was just following the format of the other elements in the plugin. do you want me to change it?

Comment (mrclay) [inline views/default/object/plugin_project/search.php]: Yes, new code at least should follow better practices.

Comment (propertunist) [inline views/default/object/plugin_project/search.php]: ok, i added another commit: https://github.com/Elgg/community_plugins/pull/147/commits/ddc7255bff759a4cd61b04ad1e0cd0b1eca77441

**Comment by mrclay** [comment]
Great! I'll test this soon

**Comment by hypeJunction** [comment]
This doesn't work

**Pull request closed without merging**